google/cloud/storage/blob.py

```diff
@@ -460,6 +463,13 @@ def generate_signed_url(
             (Optional) If true, then construct the URL relative the bucket's
             virtual hostname, e.g., '<bucket-name>.storage.googleapis.com'.
 
+        :type use_cname: bool
+        :param use_cname:
+            (Optional) If true, then construct the URL relative the bucket-bound hostname,
+            pass ``bucket-bound hostname`` value as argument of ``api_access_endpoint``,
+            e.g., 'api_access_endpoint = <bucket-bound hostname>'. See:
+            https://cloud.google.com/storage/docs/request-endpoints#cname
```

Comment:
It might be be more intuitive like this:
```
            (Optional) If true, then construct a URL corresponding to the bucket-bound
            hostname and assign its value to the ``api_access_endpoint`` argument, e.g.,
            'api_access_endpoint = <bucket-bound hostname>'. See also:
            https://cloud.google.com/storage/docs/request-endpoints#cname
```